src/python/grapl-tests-common/grapl_tests_common/clients/graphql_endpoint_client.py

```diff
@@ -17,20 +19,23 @@ class GraphQLException(Exception):
 
 class GraphqlEndpointClient:
     def __init__(self, jwt: str) -> None:
-        hostname = os.environ["GRAPL_GRAPHQL_HOST"]
-        port = os.environ["GRAPL_GRAPHQL_PORT"]
-        self.endpoint = f"http://{hostname}:{port}"
+        self.endpoint = endpoint_url("/graphQlEndpoint")
         self.jwt = jwt
 
     def query(
         self, query: str, variables: Optional[Dict[str, Any]] = None
     ) -> Dict[str, Any]:
         resp = requests.post(
-            f"{self.endpoint}/graphQlEndpoint/graphql",
+            f"{self.endpoint}/graphql",
             params={"query": query, "variables": json.dumps(variables or {})},
             cookies={"grapl_jwt": self.jwt},
         )
-        assert resp.status_code == HTTPStatus.OK, resp.json()
+        if resp.status_code != HTTPStatus.OK:
+            resp_str = "\\n".join(resp.iter_lines(decode_unicode=True))
+            raise AssertionError(
```

Comment:
feels like the wrong exception class, but also it's test code so Whatever

Reply:
The `assert` call this code replaces (to correct the issue with `resp.json()` not working as intended) raises `AssertionError` when it fails.